ios/browser/api/bookmarks/brave_bookmarks_api.h

```diff
@@ -86,6 +86,7 @@ OBJC_EXPORT
 - (void)moveToParent:(nonnull IOSBookmarkNode*)parent;
 - (void)moveToParent:(nonnull IOSBookmarkNode*)parent index:(NSUInteger)index;
 - (NSInteger)indexOfChild:(nonnull IOSBookmarkNode*)child;
+- (bool)hasAncestor:(nonnull IOSBookmarkNode*)child;
```

Comment:
Not an actual issue but just fyi you have `NS_ASSUME_NONNULL_BEGIN` wrapping the contents of this file, so these `nonnull`'s are redundant :)